### DISCUSSION TITLE
SeleniumBase on Linux Machine

### DISCUSSION THREAD

**Discussion by Orbiszeus** [seleniumbase/SeleniumBase#3002] [General]
Dear Michael,
I was using SeleniumBase on MacOS, but now I am getting a Linux machine that I will be running the crawlers on. I was wondering which version of **chrome/chrome-stable** I must be downloading to the system, does version make any difference? Also, do I have to change the code like this needs a revision with Linux:
```
def g_crawler(url, is_area):
    menu_items = []
    if not is_area: 
        with SB(uc=True, headless=True) as sb:
            # sb_config.no_sandbox = True
            sb.driver.uc_open_with_reconnect(url, 10)
```

**Comment by mdmintz** [comment]
You'll need `google-chrome-stable` or `google-chrome`. (Not `chromium`)

Also, don't use `headless` mode with `uc` mode. (That'll prevent `PyAutoGUI` from working).
Instead, use `xvfb=True`, (which may already be enabled by default for UC Mode on Linux).
